chainerx_cc/chainerx/routines/math.cc

```diff
@@ -575,6 +577,47 @@ Array Sqrt(const Array& x) {
     return out;
 }
 
+Array ElementwisePower(const Array& x1, const Array& x2) {
```

Comment:
How about using a simpler and more consistent name such as `Pow`?

Reply:
Yes, I'll change it.